server/responses/index.js

```diff
@@ -102,6 +102,14 @@ const responseList = {
   goodChallenges: {
     status: 200,
     message: 'The retrieval of challenges was successful'
+  },
+  goodUserData: {
+    status: 200,
+    message: 'The user was successfully retrieved'
+  },
+  badUserData: {
```

Comment:
Is it possible to provide a more in-depth reason for why it failed to retrieve? badUserData sounds like the user does not exist